### PR TITLE
feat: implement talosctl dashboard command

### PR DESCRIPTION
**Pull request by smira**
This builds a simple CLI UI for Talos cluster monitoring.

Some new APIs were added for monitoring based on Prometheus procfs
package.

Signed-off-by: Andrey Smirnov <smirnov.andrey@gmail.com>

---
This change is [Reviewable](https://reviewable.io/reviews/talos-systems/talos/2309)

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -1052,14 +1042,12 @@ k8s.io/apimachinery v0.0.0-20191004115801-a2eda9f80ab8/go.mod h1:llRdnznGEAqC3Dc
 k8s.io/apimachinery v0.0.0-20191109100837-dffb012825f2/go.mod h1:+6CX7hP4aLfX2sb91JYDMIp0VqDSog2kZu0BHe+lP+s=
 k8s.io/apimachinery v0.0.0-20191111054156-6eb29fdf75dc/go.mod h1:+6CX7hP4aLfX2sb91JYDMIp0VqDSog2kZu0BHe+lP+s=
 k8s.io/apimachinery v0.17.0/go.mod h1:b9qmWdKlLuU9EBh+06BtLcSf/Mu89rWL33naRxs1uZg=
-k8s.io/apimachinery v0.18.2 h1:44CmtbmkzVDAhCpRVSiP2R5PPrC2RtlIv/MoB8xpdRA=
```
Comment (smira) [inline go.sum:1055]: ran `go mod tidy` :shrug:

**Comment by smira** [comment]
current status: https://asciinema.org/a/ybt1QSDSz1MOczpf8ifUAZ7BD

File: go.sum (hunk 1 above)
Comment (andrewrynhard) [reply go.sum:1055]: Strange.

**Comment by smira** [comment]
Current version: https://asciinema.org/a/pW6H7XMYOMquDVXjXNcSkMyEA

**Comment by smira** [comment]
TODO:

- [ ] expose update interval as cli flag
- [ ] default update interval -> 3s

**Comment by smira** [comment]
Final version:  https://asciinema.org/a/97iha0qvqOJRKeg7YYVheh2WU

File: internal/app/machined/internal/server/v1alpha1/v1alpha1_server.go [hunk 2]
```diff
@@ -1048,24 +1048,25 @@ func (s *Server) Processes(ctx context.Context, in *empty.Empty) (reply *machine
 	)
 
 	for _, proc := range procs {
+		// due to race condition, reading process info might fail if process has already terminated
 		command, err = proc.Comm()
 		if err != nil {
-			return nil, err
+			continue
```
Comment (smira) [inline internal/app/machined/internal/server/v1alpha1/v1alpha1_server.go:1054]: this should fix that bug with empty output from `talosctl processes`, not perfect, but I don't know better way

**Pull request merged**